scripts/ci/sizes.py

```diff
@@ -9,12 +9,10 @@
     python3 scripts/ci/sizes.py --help
 
     python3 scripts/ci/sizes.py measure \
-        "Wasm (release)":web_viewer/re_viewer_bg.wasm \
-        "Wasm (debug)":web_viewer/re_viewer_debug_bg.wasm
+        "Wasm (release)":web_viewer/re_viewer_bg.wasm
 
     python3 scripts/ci/sizes.py measure --format=github \
-        "Wasm (release)":web_viewer/re_viewer_bg.wasm \
-        "Wasm (debug)":web_viewer/re_viewer_debug_bg.wasm
+        "Wasm (release)":web_viewer/re_viewer_bg.wasm
```

Comment:
This is a bit misleading, since we can't know if `re_viewer_bg.wasm` is the release or debug build here

Reply:
Make sure the CI sizes.py is measuring the right wasm

Reply:
The CI up until now was measuring the release size, so it should be fine

Reply:
![image](https://github.com/rerun-io/rerun/assets/1665677/5cf2cf82-0d2a-49df-be1e-792f3aee4a09)

Reply:
the `web_viewer` artifact it downloads is this one from `reusable_build_web`
![image](https://github.com/rerun-io/rerun/assets/1665677/a4c1af85-3d88-4457-aceb-c862b06c0c2f)